src/scss/components/email/_index.scss

```diff
@@ -0,0 +1,116 @@
+@import "components/form/config";
+
+hx-email-control {
+  $ctrl-columns: auto 1fr auto;
+  $ctrl-rows: auto auto;
+  $ctrl-row-align: center;
+
+  -ms-grid-columns: $ctrl-columns;
+  -ms-grid-rows: $ctrl-rows;
+  align-items: $ctrl-row-align;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    "label  label   label"
+    "prefix control suffix";
```

Comment:
### The grid needs to be updated to account for `Help Text` and `Error Message`:

![Screen Shot 2020-03-04 at 11 42 04 AM](https://user-images.githubusercontent.com/10120600/75909779-53687b80-5e12-11ea-9d16-acb765610be2.png)

Reply:
Changes have been done, please review.